README.md

```diff
@@ -36,10 +38,37 @@ install as root:
 
     sudo npm install -g @sentry/cli --unsafe-perm
 
-Or homebrew:
+By default, this package will download sentry-cli from
+[releases](https://github.com/getsentry/sentry-cli/releases). This should work
+fine for most people. If you are experiencing issues with downloading from
+GitHub, you may need to use a different download mirror. To use a custom CDN,
+set the npm config property `sentrycli_cdnurl`. The downloader will append
+`"/<version>/sentry-cli-<dist>"`.
+
+```sh
+npm install @sentry/cli --sentrycli_cdnurl=https://mymirror.com/path
```

Comment:
can we change `mymirror.com` to `mymirror.invalid` or something similar?

Reply:
done